sbncode/LArG4/CMakeLists.txt

```diff
@@ -31,6 +31,20 @@ cet_build_plugin(MergeSimSourcesSBN art::EDProducer
   BASENAME_ONLY
 )
 
+cet_build_plugin(SimpleMerge art::EDProducer
+  LIBRARIES
+  lardataobj::MCBase
+  larcorealg::headers
+  art::Framework_Principal
+  art::Framework_Services_Registry
+  art::Persistency_Common
+  messagefacility::MF_MessageLogger
+  canvas::canvas
+  fhiclcpp::types
+  fhiclcpp::fhiclcpp
+  BASENAME_ONLY
```

Comment:
`BASENAME_ONLY` should not be used. It makes the library name short (`libSimpleMerge_module.so`), which is nice if you need to type it but a nightmare when some other library with the same name happens to exist anywhere.
I **recommend** to remove it.

Reply:
done!